src/pages/iou/IOUCurrencySelection.js

```diff
@@ -0,0 +1,239 @@
+                currencySymbol: this.props.currencyList[currencyCode].symbol,
+            },
+        });
+    }
+
+    /**
+     * Returns the key used by the list
+     * @param {Object} option
+     * @return {String}
+     */
+    extractKey(option) {
+        return option.currencyCode;
+    }
+
+    /**
+     * Function which renders a row in the list
+     *
+     * @param {Object} params
+     * @param {Object} params.item
+     *
+     * @return {Component}
+     */
+    renderItem({item, key}) {
+        return (
+            <OptionRow
+                key={key}
+                mode="compact"
+                option={item}
+                onSelectRow={() => this.toggleOption(item.currencyCode)}
+                isSelected={item.currencyCode === this.state.selectedCurrency.currencyCode}
+                showSelectedState
+                hideAdditionalOptionStates
+            />
+        );
+    }
+
+    render() {
+        const sections = this.getSections();
+        return (
+            <ScreenWrapper>
+                <HeaderWithCloseButton
+                    title="Select a currency"
+                    onCloseButtonPress={() => Navigation.goBack()}
+                />
+                <View style={[styles.flex1, styles.w100]}>
+                    <View style={[styles.flex1]}>
+                        <View style={[styles.ph5, styles.pv3]}>
+                            <TextInputWithFocusStyles
+                                styleFocusIn={[styles.textInputReversedFocus]}
+                                ref={el => this.textInput = el}
+                                style={[styles.textInput]}
+                                value={this.state.searchValue}
+                                onChangeText={(searchValue = '') => {
+                                    const {currencyOptions} = getCurrencyListForSections(
+                                        this.getCurrencyOptions(this.props.currencyList),
+                                        searchValue,
+                                    );
+                                    this.setState({
+                                        searchValue,
+                                        currencyData: currencyOptions,
+                                    });
+                                }}
+                                placeholder="Search"
+                                placeholderTextColor={themeColors.placeholderText}
+                            />
+                        </View>
+                        <View style={[styles.flex1]}>
+                            <SectionList
+                                bounces={false}
+                                indicatorStyle="white"
+                                keyboardShouldPersistTaps="always"
+                                showsVerticalScrollIndicator={false}
+                                sections={sections}
+                                keyExtractor={this.extractKey}
```

Comment:
I think this is kinda extra and creates an extra level of indirection; you could just directly pass:
```suggestion
                                keyExtractor={option => option.currencyCode}
```
as an anonymous function, and get rid of `extractKey`.